src/Compilers/CSharp/Portable/Parser/LanguageParser.cs

```diff
@@ -9458,10 +9458,7 @@ internal BracketedArgumentListSyntax ParseBracketedArgumentList()
                 return (BracketedArgumentListSyntax)this.EatNode();
             }
 
-            SyntaxToken openToken, closeToken;
-            SeparatedSyntaxList<ArgumentSyntax> arguments;
-            ParseArgumentList(out openToken, out arguments, out closeToken, SyntaxKind.OpenBracketToken, SyntaxKind.CloseBracketToken);
-
+            ParseArgumentList(out var openToken, out var arguments, out var closeToken, SyntaxKind.OpenBracketToken, SyntaxKind.CloseBracketToken);
```

Comment:
>ParseArgumentList(out var openToken, out var arguments, out var closeToken, SyntaxKind.OpenBracketToken, SyntaxKind.CloseBracketToken); [](start = 12, length = 135)

Please spell out types.

Reply:
Recorded in https://github.com/dotnet/roslyn/issues/23222